deps.edn

```diff
@@ -236,7 +236,8 @@
     djblue/portal                {:mvn/version "0.51.0"}                    ; ui for inspecting values
     hashp/hashp                  {:mvn/version "0.2.2"}                     ; debugging/spying utility
     io.github.camsaul/humane-are {:mvn/version "1.0.2"}
-    io.github.metabase/hawk      {:sha "59cba703e757bd94c0cc698dbf5338bf36276991"}
+    io.github.metabase/hawk      {:git/url "https://github.com/metabase/hawk.git"
+                                  :sha     "490c9b04fbf5861dbd47c0c5281dad2d113bd285"}
```

Comment:
The upgrade is needed for to support [per test tag exclusions](https://github.com/metabase/hawk/pull/21)

I'm not sure why the git URL is required to resolve it now, while not being so before.